app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -99,16 +161,23 @@
 
   <script type="text/javascript">
     $(function () {
-      var url = "<%= preview_admin_api_docs_service_path(@api_docs_service, format: :json)-%>";
+      var host = "<%= @api_docs_service.host_with_port %>";
+      var spec = <%= ActionController::Base.helpers.sanitize(@api_docs_service.body) %>;
```

Comment:
It should work only with `sanitize @api_docs_service.body`

Reply:
What do you mean?

Reply:
He means like this I think:
```suggestion
      var spec = <%= sanitize @api_docs_service.body %>;
```
My bad btw, I suggested it like this without realising about this detail.